app/src/main/java/org/p2p/wallet/deeplinks/AppDeeplinksManager.kt

```diff
@@ -0,0 +1,76 @@
+package org.p2p.wallet.deeplinks
+
+import android.app.ActivityManager
+import android.content.Context
+import android.content.Intent
+import androidx.core.content.getSystemService
+import org.p2p.wallet.R
+import org.p2p.wallet.notification.NotificationType
+import org.p2p.wallet.root.RootActivity
+import org.p2p.wallet.utils.toStringMap
+
+class AppDeeplinksManager(private val context: Context) {
+
+    companion object {
+        const val NOTIFICATION_TYPE = "eventType"
+        const val DEEPLINK_MAIN_SCREEN_EXTRA = "DEEPLINK_SCREEN_EXTRA"
+    }
+
+    var mainTabsSwitcher: MainTabsSwitcher? = null
+
+    private var pendingIntent: Intent? = null
```

Comment:
instead of pending intent we can serialize and save a separate object inside shared prefs, and get it and use it inside the root activity

Reply:
why do you think it will be better?
also [AppDeeplinksManager.kt] created to localise all deeplinks work here

Reply:
it's just a separate way of doing deeplinks, just FYI, maybe you didn't know

Reply:
it is just a temp var for app before all needed elements will be initialised 

lets discuss it on call